Recruit-Training/Telemetry-Recruit-Training/README.md

```diff
@@ -0,0 +1,36 @@
+= U of C Solar Car Telemetry Recruit Training
```

Comment:
Viewing this on the rich viewer has some formatting issues. Turn this to an asciidoc & use a sublime previewer or https://asciidoclive.com/ to see how it looks!